### PR TITLE
Add Ukrainian locale

### PR DESCRIPTION
**Pull request by AlexanderAkhtyrtsev**
Added Ukrainian localization

### CODE REVIEW

File: src/plugins/veevalidate.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { ValidationProvider, ValidationObserver } from 'vee-validate'
 import en from '../locales/en'
 import es from '../locales/es'
 import cn from '../locales/cn'
+import ua from '../locales/ua'
+import fr from '../locales/fr'
```
Comment (davellanedam) [inline src/plugins/veevalidate.js:8]: Are you also contributing French locale?

File: src/plugins/veevalidate.js [hunk 2]
```diff
@@ -30,7 +32,9 @@ extend('url', {
 localize({
   en,
   es,
-  cn
+  cn,
+  ua,
+  fr
```
Comment (davellanedam) [inline src/plugins/veevalidate.js:37]: Same here

**Review by davellanedam — CHANGES_REQUESTED** [review]

File: src/plugins/veevalidate.js (hunk 1 above)
Comment (AlexanderAkhtyrtsev) [reply src/plugins/veevalidate.js:8]: This line fixes validation issue - translation messages does not displaying properly in French

File: src/plugins/veevalidate.js (hunk 2 above)
Comment (AlexanderAkhtyrtsev) [reply src/plugins/veevalidate.js:37]: So you could see the bug if you select French localization and fill a form's field with incorrect value

File: src/plugins/veevalidate.js (hunk 1 above)
Comment (davellanedam) [reply src/plugins/veevalidate.js:8]: But we don’t have French locales in the repo

File: src/plugins/veevalidate.js (hunk 2 above)
Comment (davellanedam) [reply src/plugins/veevalidate.js:37]: We don’t have French locale in the repo

Comment (AlexanderAkhtyrtsev) [reply src/plugins/veevalidate.js:37]: image

But I see it is already in master branch

Comment (davellanedam) [reply src/plugins/veevalidate.js:37]: Ooops! My bad!

**Comment by davellanedam** [comment]
Thanks for your contribution!

**Review by davellanedam — APPROVED** [review]

**Pull request merged**